Documentation/Signoffs/Power_System_Receiver.md

```diff
@@ -1,37 +1,148 @@
+## ESP8266 WiFi Module
+The ESP8266 can be powered through VIN pin at 3.3 V. This component allows the receiver system to connect to WiFi and communicate with the database system. The decision to use this component was decided by the receiver system design team.<br />
+## Waveshare Solar Power Management Module
+The Waveshare Solar Power Management Module is a key component for the subsystem's design, since all the other electrical components interface with it. The module uses a 14500 2500 mAh 3.7 V Li-Ion battery as the source. That battery can be recharged by solar panels ranging from 6 - 24 V and a Micro-USB port, which will be helpful for testing purposes. The module also has flexibility for output voltages with a USB-A output port, 5 V output terminal, and a 3.3 V terminal. This module's flexibility allows the design to save money, since it is capable of completing multiple functions alongside its inexpensive price. The Waveshare module also has a variety of safety features such as over-charge, over-discharge, reverse protection, over heat, and over current protections that will prevent accidents and damage to the system. It also utilizes Maximum Power Point Tracking to maximize the efficiency of the solar panel. This product was chose by the design team because of the functionality provided, because helps reduce design time and simplifies the design by serving many purposes to the system with a single component. It was also important to find a cheap product, because of the need to power nine receiver systems in different locations across campus. One downside of using this module is the complexity of the electronics makes it very hard to accurately depict its functionality in a simulation. To combat this, the design team will use values that are slightly higher than expected in our calculations to account for losses or unforseen circumstances.
+<br />
+## 14500 Rechargeable Battery 2500mAh 3.7Volt Lithium-ion Battery
+The 14500 Rechargeable Battery 2500mAh 3.7Volt Lithium-ion Battery was chosen for this project to interface with the Waveshare Solar Power Management Module. The Waveshare module only uses 14500 batteries, and this battery was chose for its high capacity at a similar price point to competing components. Calculations for the battery life is important to consider for the system. We will consider the low, typical, and high power cases to see how long the battery will last without external generation.
+<br /> <img width="578" alt="Screenshot 2024-09-16 at 9 22 25 PM" src="https://github.com/user-attachments/assets/9a4c8b51-c4b8-44f9-8ef9-9b63ab78001f"> .<br />
+
```

Comment:
The use of mixed units in the equations may be confusing to the reader. Suggestion: convert everything to the same units, e.g. mA and mAh.

Reply:
Replaced A with mA to make units more clear.

Reply:
The first equation should also be changed to show consistent units. The others are OK.

Reply:
Units changed